pkg/pool-stable/contracts/StablePool.sol

```diff
@@ -84,7 +84,10 @@ contract StablePool is BaseGeneralPool, StableMath {
 
         _require(amplificationParameter >= _MIN_AMP, Errors.MIN_AMP);
         _require(amplificationParameter <= _MAX_AMP, Errors.MAX_AMP);
-        _setAmplificationData(Math.mul(amplificationParameter, _AMP_PRECISION));
+
+        uint256 initialAmp = Math.mul(amplificationParameter, _AMP_PRECISION);
+        _setAmplificationData(initialAmp);
+        emit AmpUpdateStopped(initialAmp);
```

Comment:
Nit: We can extract a few lines from `stopAmplificationParameterUpdate` into an internal method and re-use it from here

Reply:
You're right, I don't know why I didn't do that.